service/src/test/java/uk/gov/hmcts/reform/fpl/service/CafcassEmailContentProviderTest.java

```diff
@@ -24,21 +28,35 @@
 import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
 
 @ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, CafcassEmailContentProvider.class,
+    DateFormatterService.class, HearingBookingService.class})
 class CafcassEmailContentProviderTest {
 
     private static final String LOCAL_AUTHORITY_CODE = "example";
     private static final String CAFCASS_NAME = "Test cafcass";
     private static final String COURT_EMAIL_ADDRESS = "FamilyPublicLaw+test@gmail.com";
 
-    @Mock
+    @MockBean
     private CafcassLookupConfiguration cafcassLookupConfiguration;
 
-    @Mock
+    @MockBean
     private LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
 
-    @InjectMocks
+    @Autowired
+    private ObjectMapper mapper;
+
+    private final DateFormatterService dateFormatterService = new DateFormatterService();
+    private final HearingBookingService hearingBookingService = new HearingBookingService();
+
```

Comment:
given your adding both _HearingBookingService_ and _DateFormatterService_ to _ContextConfiguration_ why are we instantiating instead of _@Autowired_ ?

Reply:
I'll use @Autowired instead 👍